src/components/MagicMarkdown.js

```diff
@@ -309,6 +309,17 @@ const OuterMarkdown = ({ markdown, limited }) => {
             </Card>
           );
         }
+        if (section.startsWith('>>>')) {
+          console.log(section);
```

Comment:
Sorry to jump in here as I'm not a reviewer, but should probably remove the console output.

Reply:
You're right, I just removed it.